contrib/log/slog/slog.go

```diff
@@ -28,6 +31,16 @@ type group struct {
 	attrs []slog.Attr
 }
 
+type config struct {
```

Comment:
nit: Maybe I'm missing some context/Go magic, but what is the reason for having a struct to hold only `log128bits`? Do we anticipate adding more values here later?

Reply:
I was following the convention found in other contribs (that is: storing configuration information under a `config` object)